src/com/company/BankAccount.java

```diff
@@ -0,0 +1,106 @@
+package com.company;
+import java.util.Scanner;
+
+public class BankAccount {
+    String userName = "Abbey";
+    int balance = 0;
+    int customerId = 112233;
+    String userChoice;
+
+
+    public void showMenu() {
+
+
+        System.out.println("Welcome " + userName + "!" + "\nYour customerID is : " + customerId);
+
+
+        do {
+
+            System.out.println("Press 1 to check balance.");
+
+            System.out.println("Press 2 to deposit funds.");
+
+            System.out.println("Press 3 to withdraw funds.");
+
+            System.out.println("Press 4 to exit menu. ");
```

Comment:
I would have this say something like "exit application" so the user knows the app will close